pkg/operator/controllers/subnets/subnet_controller_test.go

```diff
@@ -116,11 +118,11 @@ func TestReconcileManager(t *testing.T) {
 				}, nil)
 
 				subnetObjectMaster := getValidSubnet()
-				mock.EXPECT().Get(gomock.Any(), subnetResourceIdMaster).Return(subnetObjectMaster, nil).MaxTimes(2)
+				mock.EXPECT().Get(gomock.Any(), vnetResourceGroup, vnetName, subnetNameMaster, nil).Return(armnetwork.SubnetsClientGetResponse{Subnet: *subnetObjectMaster}, nil).MaxTimes(2)
 
 				subnetObjectWorker := getValidSubnet()
-				subnetObjectWorker.NetworkSecurityGroup.ID = pointerutils.ToPtr(nsgv1NodeResourceId)
-				mock.EXPECT().Get(gomock.Any(), subnetResourceIdWorker).Return(subnetObjectWorker, nil).MaxTimes(2)
+				subnetObjectWorker.Properties.NetworkSecurityGroup.ID = pointerutils.ToPtr(nsgv1NodeResourceId)
+				mock.EXPECT().Get(gomock.Any(), vnetResourceGroup, vnetName, subnetObjectWorker, nil).Return(armnetwork.SubnetsClientGetResponse{Subnet: *subnetObjectWorker}, nil).MaxTimes(2)
```

Comment:
```suggestion
				mock.EXPECT().Get(gomock.Any(), vnetResourceGroup, vnetName, subnetNameWorker, nil).Return(armnetwork.SubnetsClientGetResponse{Subnet: *subnetObjectWorker}, nil).MaxTimes(2)
```

Although it isn't making this test case to fail (because it's a disabled case maybe?), it looks like that param needs a change for the test to remain valid should anything else change in the future.

Reply:
Thanks for raising that. Will commit to update it.